test/Groups/conjugation.jl

```diff
@@ -33,19 +33,19 @@
   cc = conjugacy_class(G,x)
 
   @test length(cc) == 3
-  @test Set(elements(cc)) == Set([x^y for y in G])
+  @test Set(collect(cc)) == Set([x^y for y in G])
   y = rand(cc)
-  @test y in elements(cc)
+  @test y in collect(cc)
   @test order(y) == 2
 
   C = conjugacy_classes(G)
   @test length(C) == 5
   @test cc in C
   @test sum([length(c) for c in C]) == order(G)
-  @test sum([x in elements(c) for c in C]) == 1          # x belongs to a unique conjugacy class
-  @test sum([y in elements(c) for c in C]) == 1          # x belongs to a unique conjugacy class
+  @test sum([x in collect(c) for c in C]) == 1          # x belongs to a unique conjugacy class
+  @test sum([y in collect(c) for c in C]) == 1          # x belongs to a unique conjugacy class
```

Comment:
Why not directly `y in c`? Or doesn't it work (then we should implement it?).

Also this abuses `sum` a bit... how about this (also applied elsewhere):
```suggestion
  @test count(c -> y in c, C) == 1          # x belongs to a unique conjugacy class
```

Reply:
Yes, surely better.